docs/articles/documentation/test-api/test-code-structure.md

```diff
@@ -145,8 +148,15 @@ If the start page is not specified, it defaults to `about:blank`.
 
 ## Initialization and Clean-Up
 
-You can provide initialization code that will be executed before each test starts and clean-up code that will executed after the test finishes.
-To do this, add the `beforeEach` and `afterEach` *hook functions* to the [fixture declaration](#fixtures).
+TestCafe allows you to specify functions that will be executed before a fixture or test is started and after it is finished.
+These functions are called *hook functions* or *hooks*.
+
+### Test Hooks
+
+Test hooks are executed before a test is started and after it is finished.
```

Comment:
We should mention that hooks run for each *test run* of the test, so if test runs in 3 browsers `before` hooks will be executed 3 times: once for each browser.